### ISSUE TITLE
Cannot select a date range where start and end are same day

### ISSUE THREAD

**Issue by mollybsmith-noaa** [NOAA-GSL/MATS#91]
---

Author Name: **bonny.strong** (@bonnystrong)
Original Redmine Issue: 26267, https://vlab.ncep.noaa.gov/redmine/issues/26267
Original Date: 2016-11-11

---

If try to select a date range where end date is same date as start but different hour, gives error.

 matsMethods.getGraphData from plot_list.js : error: Error: Internal server error [500] 
c._callback (https://mats-dev1.gsd.esrl.noaa.gov/wfip2/4394873e698266c673a0571ef9dc94c83b4a8990.js?meteor_js_resource=true:3:7449) 

(was using 10/21/16:0 to 10/21/16:7)

Version 1.2.2016/11/11/06:38

**Comment by mollybsmith-noaa** [comment]
---

Original Redmine Comment
Author Name: **bonny.strong** (@bonnystrong)
Original Date: 2016-12-23T00:12:04Z

---

Confirmed that this is still an issue in MATS version 1.2.2016/12/22/20:17.
However, lowering priority to Normal.

**Comment by mollybsmith-noaa** [comment]
---

Original Redmine Comment
Author Name: **kevin.bruhwiler** (kevin.bruhwiler)
Original Date: 2017-01-27T03:19:08Z

---

This is still an issue, however it no longer throws any errors. It simply spins/loads forever.

**Comment by mollybsmith-noaa** [comment]
---

Original Redmine Comment
Author Name: **bonny.strong** (@bonnystrong)
Original Date: 2017-01-27T23:31:42Z

---

When I retested in Upper AIr Time Series, it came back ok but did not pick up all points in should have in the range I specified.  I specified 2016-12-28 00:00 to 2016-12-28 23:00 (FIM or GFS, global region, flen=12, winds).  There should have been points at 00:00 and 12:00, but it only showed the point 00:00.
